lib/kotlin/build.gradle.kts

```
@@ -34,15 +36,15 @@ dependencies {
     testImplementation(kotlin("test"))
 }
 
-kotlin {
-    jvmToolchain {
-        (this as JavaToolchainSpec).languageVersion.set(JavaLanguageVersion.of(8))
-    }
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
```

I think we still want to set the toolchain for now, to make sure that an appropriate JDK is available.

```suggestion
kotlin {
    jvmToolchain(17)
```

---

ok, seems with the current configuration i can set the toolchain to 17 and all targets to 8 :+1: